inc/msp/Client.hpp

```diff
@@ -126,111 +30,49 @@ enum MessageStatus {
 };
 
 struct ReceivedMessage {
-    uint8_t id;
-    std::vector<uint8_t> data;
+    uint32_t id;
```

Comment:
Why is this a 32bit ID? MSP2 has 16bit IDs.

Reply:
I should actually probably make it a proper msp::ID type